SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -235,8 +215,28 @@ void ActorSystem::Advance()
 		}
 	}
 
-	CreateEntityHandler.ProcessOps(*SubView->GetViewDelta().WorkerMessages);
-	ClaimPartitionHandler.ProcessOps(*SubView->GetViewDelta().WorkerMessages);
+	for (const EntityDelta& Delta : TombstoneSubView->GetViewDelta().EntityDeltas)
+	{
+		if (Delta.Type == EntityDelta::ADD || Delta.Type == EntityDelta::TEMPORARILY_REMOVED)
+		{
+			AActor* EntityActor = TryGetActor(
+				UnrealMetadata(ActorSubView->GetView()[Delta.EntityId]
```

Comment:
If I understand correctly, `ActorSubView->GetView()` will give us the same view as `TombstoneSubView->GetView()`. Should we go for the latter just in case this assumption changes in the future?

Reply:
yes! great catch, that's just a bug, my bad. Thanks!